package.json

```diff
@@ -9,9 +9,10 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "make test",
-    "ci": "make coverage",
-    "coveralls": "cat coverage/lcov.info | coveralls"
+    "test": "npm run test:unit && npm run test:integration",
+    "test:unit": "mocha --ui bdd test/*-test.js",
+    "test:integration": "mocha --ui bdd test/integration/*-test.js",
+    "posttest": "npm run lint"
```

Comment:
Removed `Makefile` and added new scripts `test:unit` and `test:integration` here